airflow/contrib/hooks/gcp_computer_hook.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
Filename is `gcp_computer_hook`. A typo. Should be `gcp_compute_hook`

Reply:
Right, thank you!